### PR TITLE
Moved `name` config option from Keystone to Admin UI

### PR DESCRIPTION
**Pull request by Vultraz**
As says, since it's no longer used by the KS core and only passed to the Admin UI.

### CODE REVIEW

**Comment by timleslie** [comment]
Haha yep, this is exactly where I was going with the changes down in the adapter code.

I'm just pondering the timing of this change, and whether we want to bundle it with this big batch of `major` changes, or wait for one more cycle. Let me have a think about it. I'm 💯 in favour of the change in principle 👍

**Comment by Vultraz** [comment]
I think it makes sense to include with the others. It logically goes with the change to db names.

File: packages/app-admin-ui/index.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ const getWebpackConfig = require('./server/getWebpackConfig');
 
 class AdminUIApp {
   constructor({
+    name,
```
Comment (timleslie) [inline packages/app-admin-ui/index.js:14]: Can we default this to: `name = 'Keystone'`?

**Comment by Vultraz** [comment]
@timleslie done

**Comment by timleslie** [comment]
> OK, I'm board with this.

Oops, *on board*, definitely not bored! 😅

**Pull request merged**